client/my-sites/email/email-providers-stacked-comparison/email-provider-stacked-card/index.tsx

```diff
@@ -31,9 +34,9 @@ const EmailProvidersStackedCard: FunctionComponent< ProviderCard > = ( props ) =
 
 	const [ areFeaturesExpanded, setFeaturesExpanded ] = useState( false );
 
-	const isViewportSizeLowerThan1040px = useBreakpoint( '<1040px' );
+	const isViewportSizeLowerThan480px = useBreakpoint( '<480px' );
```

Comment:
I think we're going to need some affordances for tablet-size screens as well. Things are not looking so great in the 480px-650px widths.